tests/e2e/utils/markdown.js

```diff
@@ -0,0 +1,36 @@
+const markdown = require('markdown-it')
+const { getLocalFileContent } = require('./fileHelper')
+
+const md = markdown()
+
+const sanitizeRenderedContent = (content) => {
+  return content
+    .replace(/&quot;/g, '"')
+    .replace(/\n(?=<\/code><\/pre>)/g, '')
```

Comment:
hmm?

Reply:
Sorry. This file is from another branch  but pushed accidentally. Removed now